docs/source/testing/mocking.md

```diff
@@ -30,15 +30,19 @@ server.listen().then(({ url }) => {
 });
 ```
 
-> Note: If `typeDefs` has custom scalar types, `resolvers` must still contain the `serialize`, `parseValue`, and `parseLiteral` functions
+> Note: If `typeDefs` has any custom scalar types, your `resolvers` must contain the `serialize`, `parseValue`, and `parseLiteral` functions for each of those custom scalars.
```

Comment:
I think the "still" on the left was relevant, though there's much better ways to express all of this.  The point this was trying to make is that we are specifying `mocks: true` *INSTEAD* of `resolvers`, which we should probably explicitly say in the previous paragraph! Then the point here is that while `mocks` can replace object field resolvers, it can't replace custom scalar "resolvers" (which are a somewhat unrelated thing that happens to go in the same data structure). And so this note should be more like "`mocks` lets you replace your object field resolvers, but if you have custom scalar types, you still need to pass in `resolvers` instead of leaving it out" or something.